### ISSUE TITLE
Bug in Chrome editing in Medium

### ISSUE THREAD

**Issue by gregormck** [languagetool-org/languagetool-browser-addon#78]
Using Chrome Version 53.0.2785.143 (64-bit) and LanguageTool Chrome Extension 1.0.4

Tl;dr - you can't correct a mistake in medium.com using the Chrome extension

---

When correcting a (French) mistake in an article on Medium using the Chrome extension Medium gives this saving error: https://www.dropbox.com/s/6phghyu9bqaj5gl/Screenshot%202016-10-22%2014.47.43.png?dl=0

Here is what Medium says about the error: https://help.medium.com/hc/en-us/articles/222246047

In addition, the cursor loses its focus and you can't click anywhere on the text and hold the cursor position. You need to do a refresh of the page to remove the conflict, which reverts the change back to the original text.

**Comment by danielnaber** [comment]
This is a similar issue as with Facebook (#70). Unfortunately, there's no solution yet. I will disable applying suggestions on medium.com, just like for Facebook.

**Comment by danielnaber** [comment]
I'll close this as very similar to #70.
